packages/components/CHANGELOG-FIGMA-COMPONENTS.md

```diff
@@ -1,5 +1,30 @@
 # [HDS Product - Components](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?type=design&node-id=6790-10926&mode=design&t=Ps0aMGZ6F3z7bAJ4-0)
 
+## August 2nd, 2024
+
+`ApplicationState` - multiple enhancements to the `ApplicationState` including:
+
+- Added support for a media slot, which when enabled, will be added above the title, allowing yielded illustrations to occupy that space.
+- Added an `alignment` property which can be set at the root level to `left` or `center`. This adjusts the component’s text, footer actions, and media alignment.
+- The footer now supports up to three actions at once, modeled after the Button Set. Previously, the actions were justified (one all the way left and one all the way right). The three actions are now organized in accordance with our [Button Organization](/patterns/button-organization) pattern with `primary` and `secondary` grouped on the left, and `tertiary` on the right. Supported actions now include Dropdown, Button, and Standalone Link. Previously Standalone Link was the only supported option.
+
+Notable visual changes in the `ApplicationState`:
+
+- The divider has been removed to modernize the component and align with our visual standards.
+- The title text has been reduced from Display 400 to Display 300 to better fit within the page hierarchy.
+- The icon and title have been changed from `foreground/faint` to `foreground/strong`. The body text has changed from `foreground/faint` to `foreground/primary`. These changes better align with our color standards established with other components.
+
+### Breaking change
+
+The changes in the footer of the `ApplicationState` to support a Dropdown, Button, and Standalone Link results in a breaking change. To avoid overwriting in-progress design work, we recommend taking screenshots of legacy instances of the Application state prior to updating your library.
```

Comment:
Answering "What changes? What exactly breaks? How exactly to avoid a loss of content." Emphasizing per [Changelog authoring best practices](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md#breaking-changes). 

```suggestion
_Adding support for three actions within the `ApplicationState` results in a breaking change to the previous actions. Before updating the library, we recommend annotating the text and icon name (with a comment or otherwise) in files that are in progress or still being referenced by engineering._
```